starter-core/src/main/java/io/micronaut/starter/feature/buildless/Buildless.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.buildless;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.gradle.GradleFile;
+import io.micronaut.starter.build.gradle.GradlePlugin;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+import io.micronaut.starter.feature.build.BuildCacheConfiguration;
+import io.micronaut.starter.feature.build.gradle.templates.buildlessGradlePlugin;
+import io.micronaut.starter.feature.build.gradle.templates.gradleBuildCache;
+import io.micronaut.starter.template.RockerWritable;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class Buildless implements Feature, BuildCacheConfiguration {
```

Comment:
this should implement `CommunityFeature` instead of `Feature`

Reply:
@sdelamo I implemented `CommunityFeature`, but this code prevents tests from running properly:

https://github.com/micronaut-projects/micronaut-starter/blob/9c68883693ea2f82cc1e4789c4d8571ebd8ad577/starter-core/src/main/java/io/micronaut/starter/feature/CommunityFeatureValidator.java#L34-L37

So I've flipped `ENABLE_COMMUNITY_FEATURES`:
https://github.com/micronaut-projects/micronaut-starter/blob/0f3839464278b8e03b9db227d14cae60a968514e/starter-core/src/main/java/io/micronaut/starter/feature/CommunityFeatureValidator.java#L27

Tests still pass as far as I can tell; is this the right thing to do?